### ISSUE TITLE
Lance do usuário

### ISSUE THREAD

**Issue by ThiagoHBA** [pitch-a-boo-organization/pitch-a-boo-game#31]
- [ ] Quando o número do bid é igual a 10 fica impossível diminuir. 
 - [ ] O número de bid não está respeitando a quantidade de bones que o usuário tem disponível.

**Comment by JoanWilson** [comment]
Ei, preciso resolver esse aí. Esqueci total ou vocês já resolveram?
